src/components/Sykmeldingview/Sections/MedisinskVurderingView.tsx

```diff
@@ -11,25 +11,34 @@ const MedisinskVurderingView: React.FC<{ medisinskVurdering?: MedisinskVurdering
         return null;
     }
 
+    if (arbeidsgiver) {
```

Comment:
om det er arbeidsgiver kan man vel bare returnere null med en gang?

Reply:
![image](https://user-images.githubusercontent.com/25056612/121347630-b3a87000-c927-11eb-8910-a43ddc20871f.png)
Vi viser en sladdet versjon. For å gjøre det tydelig at det ikke vises, i guess

Reply:
aha, så da er det faktisk satt noe i diagnosen da?

Reply:
for det sjekkes jo i den if setningen under også? 🤔

Reply:
Hvis det finnes noen diagnose (eller bidiagnose) så vises den som "sladdet" i arbeidsgivervisning. Legg merke til propen "sladd"